nodes/scheduling/nodes-custom-scheduler.adoc

```diff
@@ -0,0 +1,27 @@
+[id="nodes-custom-scheduler"]
+= Running a custom scheduler
+include::modules/common-attributes.adoc[]
+:context: nodes-custom-scheduler
+
+toc::[]
+
+You can run multiple custom schedulers alongside the default scheduler and configure which scheduler to use for each Pod.
+
+To schedule a given Pod using a specific scheduler, xref:../../nodes/scheduling/nodes-custom-scheduler.adoc#nodes-custom-scheduler-deploying_nodes-custom-scheduler[specify the name of the scheduler in that Pod specification].
+
+[NOTE]
+====
+Each scheduler has a separate view of resources in a cluster. For that reason, each scheduler should operate over its own set of nodes.
+
+If two or more schedulers operate on the same node, they might intervene with each other and schedule more Pods on the same node than there are available resources for. Pods might get rejected due to insufficient resources in this case.
```

Comment:
I think we're avoiding ending with propositions.

Reply:
Actually the ISG allows for ending with prepositions:

> Use a preposition at the end of a sentence to avoid an awkward or stilted construction.
> Example (awkward)
> Click the item for which you want to search.
> Example (improved)
> Click the item that you want to search for.

Reply:
@bergerhoffer, ah, okay; I've been called out before, so that's what prompted my comment.